source/lib/expat_lib/xmlparse.c

```diff
@@ -1502,6 +1502,7 @@ XML_Parse(XML_Parser parser, const char *s, int len, int isFinal)
       errorCode = XML_ERROR_NO_MEMORY;
       return XML_STATUS_ERROR;
     }
+    __attribute__((fallthrough));
```

Comment:
For every file where `__attribute__` is used add the following protection:
`#ifndef(__GNUC__)
#define __attribute__(X)
#endif`